tests/framework/platforms/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load(":versions.bzl", "make_applications", "make_frameworks", "make_frameworks_with_dependencies")
+
+make_frameworks(
+    srcs = ["platform.m"],
+    resources_to_bundle = glob(["Resources/**/*"]),
+)
+
+make_frameworks_with_dependencies(
+    srcs = ["platform.m"],
+    resources_to_bundle = glob(["Resources/**/*"]),
+)
+
+make_applications(
```

Comment:
Did you confirm that this application would fail to run, if this transition was not applied?

Reply:
Yes